1.1/geo.ttl

```diff
@@ -580,6 +589,53 @@ geosparql:
 	skos:prefLabel "spatial dimension"@en ;
 .
 
+:metricSpatialResolution
+	a owl:DatatypeProperty ;
+	rdfs:domain :Geometry ;
+	rdfs:range xsd:double ;
+	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
+	skos:definition """The spatial resolution of a Geometry in meters."""@en ;
+	skos:note """Spatial resolution specifies the level of detail of a Geometry. It the smallest dinstinguishable distance between spatially adjacent coordinates."""@en;
+	skos:prefLabel "metric spatial resolution"@en ;
+.
+
+:metricSpatialAccuracy
```

Comment:
This does not follow the DCAT convention where the property name is `*InMeters`. 
However, element IDs are not as important as labels and definitions.